### ISSUE TITLE
Split jekyll theme

### ISSUE THREAD

**Issue by ocefpaf** [ioos/notebooks_demos#82]
The theme used in the `gh-pages` is a standard Jekyll web-page with the directories `_layout`, `_includes`, and `assets`. I am thinking that maybe we should put those in an "official" IOOS `jekyll` theme and pull them here as submodules. That will make it easier to use  them in other projects and change things in one place while propagating them to all pages.

What do you think @jbosch-noaa and @lukecampbell? Makes sense?

@lukecampbell any feedback on the theme is welcomed! `css`/`less` is new to me and.

**Comment by jbosch-noaa** [comment]
@ocefpaf  - That seems to make sense to me.

**Comment by ocefpaf** [comment]
Done in https://github.com/ioos/ioos_jekyll_theme/issues/4

**Comment by mwengren** [comment]
@ocefpaf I've already spent some time adapting an existing Jekyll theme to make it look IOOS-like for github.io documentation for the Catalog project.  Maybe we can combine these two efforts somehow?  I don't know what kind of look you are going for, but I found it pretty easy to adapt a theme called 'minimal-mistakes' to give it an IOOS style.  It's also pretty easy to customize in terms of the navigation menu, etc, for other projects, and has a lot of the nuts and bolts already set up in Jekyll.

Here's a link to to the working version: https://ioos.github.io/catalog/pages/registry/
